ext/pgsql/pgsql.c

```diff
@@ -605,11 +601,6 @@ PHP_MINFO_FUNCTION(pgsql)
 	php_info_print_table_start();
 	php_info_print_table_row(2, "PostgreSQL Support", "enabled");
 	php_info_print_table_row(2, "PostgreSQL (libpq) Version", pgsql_libpq_version);
-#ifdef HAVE_PGSQL_WITH_MULTIBYTE_SUPPORT
-	php_info_print_table_row(2, "Multibyte character support", "enabled");
```

Comment:
some people might be "surprised" by the disappearance of this info wdyt ?

Reply:
Sure. Otherwise, it cannot be disabled but next commit with rebase has this added back so it's more clear, yes.